src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -1898,7 +1898,7 @@ int wm_vuldet_linux_oval_vulnerabilities(sqlite3 *db, agent_software *agents_it,
             os_strdup(condition, vuln_pkg->vuln_cond->condition);
         }
 
-        vuln_pkg->feed = VU_SRC_OVAL;
+        vuln_pkg->feed |= VU_SRC_OVAL;
```

Comment:
Hi,
`vuln_pkg->feed` is set by default to 0, so I think it's unnecessary to do an  `OR` operation since the value will be exactly the same one.

Reply:
Hi, it has been added just to make the assignment independent from the default value.